subworkflows/local/taxa.nf

```diff
@@ -0,0 +1,55 @@
+/*
+    TAXONOMIC CLASSIFICATION
```

Comment:
Please file and workflow to `TAXONOMIC_CLASSIFICATION`, or if you don't like it so it's too long, `TAX_CLASS` or similar

Reply:
context: This will be important for the nf-core-level subworkflow :)

Reply:
i like TAX_CLASS.